### PR TITLE
Make layout and font-size responsive

### PR DESCRIPTION
**Pull request by antoine-boue**

### CODE REVIEW

File: assets/styles/sass/styles.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@import "shared/reset";
 @import "shared/shared";
+@import "shared/reset";
```
Comment (windresistance) [inline assets/styles/sass/styles.scss:2]: shouldn't the reset come first?

**Review by windresistance — CHANGES_REQUESTED** [review]

File: assets/styles/sass/styles.scss (hunk 1 above)
Comment (antoine-boue) [reply assets/styles/sass/styles.scss:2]: But we use the mixins in the reset file.

Comment (windresistance) [reply assets/styles/sass/styles.scss:2]: Roger that. Reset should always come first, but yeah in this case I guess we can't. Makes me think we should just combine shared and reset into one file. What do you think?

**Review by windresistance — APPROVED** [review]

File: assets/styles/sass/styles.scss (hunk 1 above)
Comment (antoine-boue) [reply assets/styles/sass/styles.scss:2]: Agreed

**Pull request merged**